### ISSUE TITLE
Error on `className` with React 19

### ISSUE THREAD

**Issue by Arkellys** [FortAwesome/react-fontawesome#560]
**Describe the bug**

On React 19 (beta), an error is thrown if `className` is not passed to `<FontAwesomeIcon>`.
I believe it's the same bug as the one reported on [this issue](https://github.com/FortAwesome/react-fontawesome/issues/347), except that now it happens even when no `className` is specified.

**Reproducible test case**

https://codesandbox.io/p/sandbox/react-19-font-awesome-k6n8ww?file=%2Fsrc%2FApp.js%3A11%2C51

Error on the sandbox:

```shell
TypeError: className is undefined
```

I get a different error when I try it on my app (maybe it depends on the bundler config):

```shell
Uncaught TypeError: Cannot read properties of undefined (reading 'split')
```

**Expected behavior**

The error is fixed when adding a `className`, but this prop should not be required.

**Setup**

- Browsers: Firefox (125.0.3), Electron (Chromium M124)
- Font Awesome `6.5.2`
- react-fontawesome `0.2.0`
- React `19.0.0-beta-6946ebe620-20240508`

**Comment by Arkellys** [comment]
I just noticed [there is already a PR](https://github.com/FortAwesome/react-fontawesome/pull/559) to fix this, hopefully it will be merged quickly.

**Comment by robmadole** [comment]
I just released 0.2.1 which I think will fix this. Can you all retest?

**Comment by sakispirtos** [comment]
i just pulled down 0.2.1 and my application is now showing this error.  i had to roll back to 0.2.0 to resolve.

**Comment by Arkellys** [comment]
> I just released 0.2.1 which I think will fix this. Can you all retest?

I can confirm that version `0.2.1` solves this issue on React 19, thank you!
